src/Set.php

```diff
@@ -92,8 +92,8 @@ public function clear()
      *
      * @param mixed ...$values
      *
-     * @return bool true if at least one value was provided and the set
-     *              contains all given values, false otherwise.
+     * @return bool false if any of the provided values are not in the set;
+     *              true otherwise.
```

Comment:
This is a correction, yes. I wonder if we should instead reason in the positive, in this case 

```
@return bool true if all values exist in the set, or none given.
```